rclcpp/src/rclcpp/clock.cpp

```diff
@@ -32,7 +32,7 @@ class Clock::Impl
   {
     rcl_ret_t ret = rcl_clock_init(clock_type, &rcl_clock_, &allocator_);
     if (ret != RCL_RET_OK) {
-      exceptions::throw_from_rcl_error(ret, "could not get current time stamp");
+      exceptions::throw_from_rcl_error(ret, "Failed to initialize rcl clock");
```

Comment:
we usually start error messages in lower case:
```suggestion
      exceptions::throw_from_rcl_error(ret, "failed to initialize rcl clock");
```

Reply:
got it! take your suggestion fix.